### PR TITLE
Fix to CompositeMap axes scaling

### LINKED ISSUE #2294: Composite map - axes in arcsec
Hello,

after the upgrade to SunPy 0.8 all my composite maps have a different axes scaling compared to the "usual" arcsec - I see that even the example image from the documentation shows that the X- and Y- axes of the full disk go from -0.3 to +0.3 arcsec. 

maps-8

How to make the plot show the "true" arcsec values? If I convert the axes values into u*arcsec nothing happens, because according to the plot the axes are still in arcsec.

### PR DESCRIPTION
**Pull request by wafels**
Issue #2294 reports that CompositeMap does not have the correct axes scaling.  I think this was caused by the default WCSAxes choice of reporting degrees when possible.  This is fixed by copying the code from mapbase that selects the correct axes scaling and including it in CompositeMap.

Fixes #2294.

### CODE REVIEW

**Comment by wafels** [comment]
By the way,  "Write unit tests for sunpy.map.compositemap" is issue #218 . Unit tests would have caught #2294.

File: sunpy/map/compositemap.py [hunk 1]
```diff
@@ -413,9 +413,13 @@ def plot(self, axes=None, annotate=True,  # pylint: disable=W0613
...
         for m in self._maps:
             # Parameters for plotting
+            bl = m._get_lon_lat(m.bottom_left_coord)
+            tr = m._get_lon_lat(m.top_right_coord)
+            x_range = list(u.Quantity([bl[0], tr[0]]).to(m.spatial_units[0]).value)
+            y_range = list(u.Quantity([bl[1], tr[1]]).to(m.spatial_units[0]).value)
             params = {
                 "origin": "lower",
-                "extent": list(m.xrange.value) + list(m.yrange.value),
+                "extent": list(x_range) + list(y_range),
```
Comment (nabobalis) [inline sunpy/map/compositemap.py]: Does this need to be listed again?

Comment (wafels) [reply sunpy/map/compositemap.py]: The second list is not necessary.

**Comment by wafels** [comment]
Not sure - looks like travis and appveyor couldn't find some packages.  temporary CI freakout???

File: sunpy/map/compositemap.py [hunk 2]
```diff
@@ -413,9 +415,13 @@ def plot(self, axes=None, annotate=True,  # pylint: disable=W0613
         # Plot layers of composite map
         for m in self._maps:
             # Parameters for plotting
+            bl = m._get_lon_lat(m.bottom_left_coord)
+            tr = m._get_lon_lat(m.top_right_coord)
+            x_range = list(u.Quantity([bl[0], tr[0]]).to(m.spatial_units[0]).value)
+            y_range = list(u.Quantity([bl[1], tr[1]]).to(m.spatial_units[0]).value)
```
Comment (Cadair) [inline sunpy/map/compositemap.py]: BUG

**Pull request merged**